src/styles/helpers/__test__/getContrastText.spec.js

```diff
@@ -0,0 +1,66 @@
+import { getContrastText } from '../color';
+import { COLOR_BRAND, COLOR_SUCCESS, COLOR_WARNING } from '../../colors';
+
+jest.mock('../color/getContrastRatio', () => background => {
+    if (background === '#fc0') {
+        return false;
+    }
+    return true;
```

Comment:
There are certain situations where you can return whatever you want from mocks but in this case, I think we should make getContrastRatio to return number values to simulate the real function.